docs/CUSTOMIZATION.md

```diff
@@ -0,0 +1,147 @@
+Customization Options
+=====================
+
+```xml
+<com.prolificinteractive.materialcalendarview.MaterialCalendarView
+    android:id="@+id/calendarView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    app:mcv_showOtherDates="boolean"
+    app:mcv_arrowColor="color"
+    app:mcv_selectionColor="color"
+    app:mcv_headerTextAppearance="style"
+    app:mcv_dateTextAppearance="style"
+    app:mcv_weekDayTextAppearance="style"
+    app:mcv_weekDayLabels="array"
+    app:mcv_monthLabels="array"
+    app:mcv_tileSize="dimension"
+    app:mcv_firstDayOfWeek="enum"
+    app:mcv_leftArrowMask="drawable"
+    app:mcv_rightArrowMask="drawable"
+    />
+```
+
+## Tile Size
+
+One of the fundamental concepts in this library is that of `tileSize`.
+Even if you don't set one explicitly, one is calculated and is used in sizing the view.
+
+By default, with no tile size set, the view will scale to fill as much space as it can.
+The functionality is similar to ImageView with `adjustViewBounds` set to true.
+
+If a tileSize is set, that will override the `layout_width` and `layout_height` set.
+
+The view is 7 tiles wide and 8 tiles high (with the top bar visible).
+
+
+## Date Selection
+
+We support three modes of selection: single, multiple, or none. The default is single selection.
+The mode can be changed by calling `setSelectionMode()` and passing the appropriate constant (`SELECTION_MODE_NONE`, `SELECTION_MODE_SINGLE`, or `SELECTION_MODE_MULTIPLE`).
+If you change to single selection, all selected days except the last selected will be cleared.
+If you change to none, all selected days will be cleared.
+
+You can set an `OnDateSelectedListener` to listen for selections, make sure to take into account multiple calls for the same date and state.
+You can manually select or deselect dates by calling `setDateSelected()`.
+Use `setSelectedDate()` to clear the current selection(s) and select the provided date.
+
+There are also: `clearSelection()`, `getSelectedDates()`, and `getSelectionMode()`; which should be self-explanitory.
```

Comment:
I like something like "follow the same paradigm" or "use the same methodology" or "work the same way" instead of self-explanatory too. It's less heavy for the reader.